.circleci/config.yml

```diff
@@ -84,17 +84,6 @@ jobs:
           at: '~/pulmap'
       # Install Bundler
       - run: gem install bundler -v '~> 2.0'
-      - run:
```

Comment:
We don't need this one anymore?

Reply:
Shouldn't it be fine to just set the node version in the build step?

Reply:
It's what we do in Pulfalight: https://github.com/pulibrary/pulfalight/blob/main/.circleci/config.yml#L15

Reply:
Maybe the Pulfalight config is a mistake.

Reply:
Yep. Still needed. Added back with new node version.